core/remote/change.js

```diff
@@ -40,11 +40,13 @@ export type RemoteChange =
   | RemoteFileUpdate
 
 export type RemoteIgnoredChange = {type: 'RemoteIgnoredChange', doc: Metadata|RemoteDoc|RemoteDeletion, detail: string}
+export type RemoteDescendantChange = {type: 'RemoteDescendantChange', doc: Metadata|RemoteDoc|RemoteDeletion, detail: string}
```

Comment:
`RemoteNoise` is not used anymore.
Subtypes should be merged with `RemoteChange` ones.
Which requires using the `sideName` property instead of a `Remote` prefix in `type` (same as `RemoteChange`).
Done in #1262.